weaver/datatype.py

```diff
@@ -821,7 +821,23 @@ def version(self):
     @property
     def inputs(self):
         # type: () -> Optional[List[Dict[str, Any]]]
-        return self.get("inputs")
+        """
+        According to `OGC-API`, ``maxOccurs`` and ``minOccurs`` representations should be:
+            ``maxOccurs``: ``int`` or ``"unbounded"``
+            ``minOccurs``: ``int``
```

Comment:
I think you will need some `- ` to make a list. 
Renders incorrectly in the doc: 
https://pavics-weaver--272.org.readthedocs.build/en/272/autoapi/weaver/datatype/index.html#weaver.datatype.Process.inputs

Reply:
Seems like the change in ee8a697 didn't work. 
You can try to run `make docs` to tests more quickly locally the rendering.

Reply:
I added the `-` before you refreshed I think that's why